src/client/index.html

```diff
@@ -7,7 +7,7 @@
     <meta charset="utf-8">
     <title>Open Energy Dashboard</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
Is there a reason why we're including fontawesome in this way when we include bootstrap via npm and webpack?